src/app/components/PrizeField/PrizeField.tsx

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import styles from "./PrizeField.module.css";
+
+type PrizeFieldProps = {
+  variant: "Primary" | "Secondary";
+};
+
+export default function PrizeField({ variant }: PrizeFieldProps): JSX.Element {
+  return (
+    <div className={styles[variant]}>
```

Comment:
This doesn't need to get the class from the variant prop as the classes contain the exact same properties